src/BasketBundle/Form/PaymentType.php

```diff
@@ -99,7 +100,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $sub = $builder->create('paymentMethod', $choiceType, array(
             'expanded' => true,
-            'choice_list' => new SimpleChoiceList($choices),
+            'choice_list' => function (Options $opts, $previousValue) use ($choices) {
```

Comment:
You can do it this way: https://github.com/sonata-project/SonataClassificationBundle/pull/280
Also I made a typo there ;) so check this too https://github.com/sonata-project/SonataClassificationBundle/pull/284